Sources/DeepLearning/Layer.swift

```diff
@@ -257,7 +332,10 @@ public extension Dense {
     }
 }
 
-/// A convolutional neural network layer.
+/// A 2D convolution layer (e.g. spatial convolution over images).
```

Comment:
Do you prefer `2-D` over `2D`?
I prefer `2-D`.

Reply:
Of course `2-D`.